docs/4.1/admin-guide.md

```diff
@@ -1991,6 +1991,9 @@ belong to:
 $ tctl users add joe --k8s-groups="system:masters"
 ```
 
+!!! tip "NOTE":
+    In order to execute the above command, a Teleport role in RBAC needs to be created as `k8s-group`. An example RBAC role can be found in the [Kubernetes SSH](kubernetes_ssh.md#impersonation) section.
```

Comment:
Is this saying you need a role specifically called `k8s-group` in Kubernetes RBAC? Referring to some kind of `ClusterRole` or `ClusterRoleBinding` that will enable impersonation inside Kubernetes? It's not incredibly clear currently.

Reply:
Reword to something along the following:

The above command is for the OSS version, Enterprise customers manage k8s-groups through RBAC

Reply:
@aelkugia Can you add this to 4.2 docs also.

Reply:
I'm just going to close this PR as it is pretty out of date, and in discussion with @fspmarshall the below update of where I was handling the error was incorrect as well. I can add updates to docs to 4.1 and 4.2 in the new PR which I will create